src/dtos/user-dto.ts

```diff
@@ -0,0 +1,19 @@
+import { User } from '../entity/user';
+
+export class UserDto {
```

Comment:
We need to identify and use the persistent ID to identify objects of this class.
We need to specify whether the login ID / eMail is either changeable or re-assignable for each IDP we want to work with.